src/contracts/xMETRIC.sol

```diff
@@ -5,14 +5,14 @@ import "@openzeppelin/contracts/access/Ownable.sol";
 import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 
 contract xMETRIC is ERC20("xMETRIC", "xMETRIC"), Ownable {
```

Comment:
`contract xMETRIC is ERC20("xMETRIC", "xMETRIC"), ERC20Pausable, Ownable  {`

And in the contract itself add two functions
`    function pause() public onlyOwner {
        _pause()
    }`
    
    ` function unPause() public onlyOwner {
        _unpause()
    }`

Reply:
since we have a totally custom transfer function I don't think we can/should rely on openzeppelin for this one - might not work out of the box

Reply:
this functionality is called in  function _beforeTokenTransfer(
which is called before _mint